sql/hive/src/main/scala/org/apache/spark/sql/hive/hiveUdfs.scala

```diff
@@ -81,6 +82,8 @@ private[hive] case class HiveSimpleUdf(funcWrapper: HiveFunctionWrapper, childre
 
   override def nullable: Boolean = true
 
+  override def checkInputDataTypes(): TypeCheckResult = TypeCheckResult.TypeCheckSuccess
```

Comment:
Can we get the required data types? If we can, we should absolutely check input types.